packages/nodes-base/nodes/RespondToWebhook/utils.ts

```diff
@@ -18,3 +22,47 @@ export const configuredOutputs = (
 
 	return ['main'];
 };
+
+export const replaceSingleQuotes = (responseBody: string) => {
+	return responseBody.replace(/'/g, '&#39;');
+};
+
+/*
+ * To prevent XSS and CSRF, we sanitize the response body by wrapping it in an iframe.
+ * This prevents the API from being accessed since the iframe origin will be empty.
+ */
+export const sanitizeResponseData = (responseBody: string, shouldReplaceSingleQuotes: boolean) => {
```

Comment:
Do we really need the `shouldReplaceSingleQuotes`? Why don't we always do the replacement?